app/models.py

```diff
@@ -114,6 +114,7 @@ class User(db.Model):
     platform_admin = db.Column(db.Boolean, nullable=False, default=False)
     current_session_id = db.Column(UUID(as_uuid=True), nullable=True)
     auth_type = db.Column(db.String, db.ForeignKey('auth_type.name'), index=True, nullable=False, default=SMS_AUTH_TYPE)
+    last_validated_at = db.Column(db.DateTime, nullable=True)
```

Comment:
Do we want to allow it to be nullable?

Reply:
Also I think it would be a good idea to give it an even more verbose name like `email_access_last_validated_at`? Just if I was new the codebase, I probably wouldn't know this was to do with their email account. You could also a code comment above here describing what this variable is trying to record/solve (government users losing access to their email but retaining their SMS device and therefore still being able to access notify).

Reply:
> Do we want to allow it to be nullable?

Hmmm you are right, we don't need it to be nullable

> Also I think it would be a good idea to give it an even more verbose name like email_access_last_validated_at

Good idea 👍